node-notes-cli/app.js

```diff
@@ -0,0 +1,52 @@
+import * as fs from 'node:fs/promises';
+
+try {
+  if (process.argv[2] === 'read') {
+    read();
+  }
+  if (process.argv[2] === 'create') {
+    create();
+  }
+  if (process.argv[2] === 'update') {
+    update();
+  }
+  if (process.argv[2] === 'delete') {
+    remove();
+  }
+} catch (err) {
+  console.error('Error:', err.message);
+}
+
+async function read() {
+  const content = await fs.readFile('data.json');
+  const info = JSON.parse(content);
+  const notes = info.notes;
+  const done = JSON.stringify(notes, null, 2);
+  console.log(done);
```

Comment:
It would be best to make your output a little more user friendly. Outputting a JSON string isn't the nicest display. Instead, loop through the notes and write each one separately.